<launch>

  <arg name="gui" default="true"/>
  
  <include file="$(find pcl_tools)/launch/pointcloud_karage.launch">
    <arg name="gui" value="false"/>
  </include>

  <node pkg="pcl_graph_cut" type="graph_cut_ros" name="graph_cut_ros">
    <param name="input_topic_name" value="karage_points"/>
    <param name="output_topic_name" value="graph_cut_points"/>
  </node>

  <group if="$(arg gui)">
	  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find pcl_graph_cut)/rviz/demo_karage.rviz" required="true" />
  </group>

</launch>
